master/main.go

```diff
@@ -1,5 +1,113 @@
 package main
 
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+
+	"./models"
```

Comment:
Don't use local imports. Code that doesn't have to be shared with other packages doesn't have to be in a separate package.